src/components/Modeler.vue

```diff
@@ -267,10 +260,8 @@ export default {
     },
     highlightedNode: () => store.getters.highlightedNode,
     invalidNodes() {
-      return Object.entries(this.validationErrors).reduce((invalidIds, [, errors]) => {
-        invalidIds.push(...errors.map(error => error.id));
-        return invalidIds;
-      }, []);
+      return Object.entries(this.validationErrors)
+        .flatMap(([, errors]) => errors.map(error => error.id));
```

Comment:
I felt this was slightly simpler to read than the reduce, which is functionally equivalent.